djangodenmark/static_files/css/skeleton.css

```diff
@@ -76,7 +76,7 @@
   .one-third.column               { width: 30.6666666667%; }
   .two-thirds.column              { width: 65.3333333333%; }
 
-  .one-half.column                { width: 48%; }
+  .one-half.column                { width: 100%; }
```

Comment:
This file should not be edited

But also, it doesn't make sense the change the `one-half` rule to not be one half :)